scripts/download-calico-crds.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
Maybe I am not getting this, when is this script executed?

Reply:
I was thinking we might have multiple scripts here. One that pulls CRDs from apiextensions for example. It keeps retagger cleaner. I can add a comment explaining the link.